android/java/org/chromium/chrome/browser/night_mode/settings/BraveThemePreferences.java

```diff
@@ -34,8 +35,9 @@ public void onCreatePreferences(@Nullable Bundle savedInstanceState, String root
         Profile mProfile = Profile.getLastUsedProfile();
         NTPBackgroundImagesBridge mNTPBackgroundImagesBridge = NTPBackgroundImagesBridge.getInstance(mProfile);
         if (!NTPBackgroundImagesBridge.enableSponsoredImages()
-            || (mNTPBackgroundImagesBridge != null
-            && !mNTPBackgroundImagesBridge.isSuperReferral())) {
+                || (mNTPBackgroundImagesBridge != null
+                    && !mNTPBackgroundImagesBridge.isSuperReferral())
+                || Build.VERSION.SDK_INT <= Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
it should be `Build.VERSION_CODES.M` I guess

Reply:
@SergeyZhukovsky it should be okay. as it's removing SR theme option for android lollipop and lower apis